### ISSUE TITLE
Any suggestions for web devicons?

### ISSUE THREAD

**Issue by pysan3** [nvim-neorg/neorg#792]
### Issues

- [X] I have checked [existing issues](https://github.com/vhyrro/neorg/issues?q=is%3Aissue) and there are no existing ones with the same request.

### Feature description

Hi, thanks for the amazing plugin!

[nvim-web-devicons](https://github.com/nvim-tree/nvim-web-devicons) is a plugin used in many plugins such as nvimtree, neo-tree, bufferline etc to provide nice looking icons for files depending on their filetype.

User can define their own icons by setting these parameters on setup. And here is an example I define myself for snippet files. It is nice to have a dedicated icon and would be easier to spot the file when many types of files are in the same directory.

```lua
require("nvim-web-devicons").setup({
  override = {
    snippets = {
      icon = "",
      color = "#DBC63E",
      name = "Snippets",
    },
  },
})
```

|Default Icons|Above Config|
|-|-|
|![Screenshot_20230409_024217](https://user-images.githubusercontent.com/41065736/230735865-00401776-55ca-4a6a-b407-912dd18b0e5f.png)|![Screenshot_20230409_025142](https://user-images.githubusercontent.com/41065736/230735899-7e22f64a-5c48-480f-bc5e-5dc6fc549f86.png)|

## Issue 

However, unfortunately `.norg` file is not defined in the look up table and the default icons is used in the file tree, which I feel a bit sad.

Therefore, I would like to ask for any suggestion for a good icon (any [nerd font](https://www.nerdfonts.com/cheat-sheet)) and color to represent norg files.

Creating an PR to web-devicons would be a good option as well.

Here's one I came up with but would love to hear alternatives.
```lua
    norg = {
      icon = "◉", -- Default icon for Header 1
      color = "#4878be",
      name = "Neorg",
    },
```

![Screenshot_20230409_030320](https://user-images.githubusercontent.com/41065736/230736335-9cd237d1-171b-46ba-9309-1ca1468f5e98.png)

### Help

No

**Comment by pysan3** [comment]
Found this

https://github.com/devicons/devicon/pull/1346

Color updated to: `#4878be`

**Comment by vhyrro** [comment]
Yep the norg icon exists officially in devicons (very epic). Closing because the issue has a solution :)

**Comment by pysan3** [comment]
Sorry I don't have much knowledge of fonts, but does that mean that it will be usable as a nerd font when it is released?

It seems I can't find it here at the moment. 

https://www.nerdfonts.com/cheat-sheet

**Comment by pysan3** [comment]
`nvim-web-devicons` is a plugin that just has a lookup table mapping file types to its icon (in nerdfont) and color. It does not bring in any icons but the terminal should be rendered with a nerd font supported font. 

So basically, when nerd font adds norg icon (I don't know how tho), we can use that instead, but my first intent of this issue was to ask for an idea of any existing nerd font icon to express a norg file. 

(For example, the scissors icon is not any _official_ icon for snippets but I'd say it matches pretty well.)

**Comment by tamton-aquib** [comment]
Heyyo I was just reading the other issue (from @noornee) regarding this, I had it temporarily working some time ago. So just wanted to share:

<img width="902" alt="image" src="https://github.com/nvim-neorg/neorg/assets/77913442/44802846-ee1a-4f20-b406-56cfd50a931b">

![image](https://github.com/nvim-neorg/neorg/assets/77913442/855b20b0-dcca-4e73-8f7d-4b5ac0393029)

The steps:
   - Upload [neorg.svg](https://github.com/nvim-neorg/neorg/blob/main/res/neorg.svg) to [icomoon](https://icomoon.io). Note the code point, make sure it doesnt clash with others.

   - Download the zip file and unzip to get the ttf file (or others if you want).

   - Add it into your font directory. (`~/.local/share/fonts/` in Linux and `~/Library/Fonts/` in mac OS)

   - Change settings in terminal.
   	- **Kitty** - `symbol_map` to map the unicode code point to the font that we downloaded and moved. Dont change any other font configurations, just map this single symbol using [symbol_map](https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.symbol_map).
   	- **Wezterm** - Add the font file as fallback. I have not done it directly but has seen others do it.
   	- Not sure about other terminals

   - Lastly add the icon to your local nvim-web-devicons setup() call. Mine looks something like:
  ```lua
require("nvim-web-devicons").setup({
	override = {
		norg = {
			icon = " ",		-- This icon probably wont be visible in your browser, but insert your icon here.
			color = "#4878be",	-- Neorgs official color
			name = "neorg"
		}
	}
 })
```

To insert a unicode codepoint in vim, you can do this **in insert mode**: `<C-v>u` followed by the point.
In my case it was E420, therefore: `<C-v>ue420` in insert mode.

Hope this helps :)

**Comment by tamton-aquib** [comment]
Also wanted to mention some related issues/discussions:

- https://github.com/ryanoasis/nerd-fonts/issues/615
- https://github.com/devicons/devicon/issues/1420
- https://github.com/ryanoasis/nerd-fonts/discussions/1095
